starport_template/lib/model/tx_json.dart

```diff
@@ -0,0 +1,59 @@
+class Tx {
+  Tx({
+    required this.type,
+    required this.body,
+  });
+
+  String type;
+  Body body;
+
+  factory Tx.fromJson(Map<String, dynamic> json) => Tx(
+        type: json["@type"] as String? ?? '',
+        body: Body.fromJson(json["body"] as Map<String, dynamic>),
+      );
+}
+
+class Body {
+  Body({required this.messages});
+
+  List<Message> messages;
+
+  factory Body.fromJson(Map<String, dynamic> json) =>
+      Body(messages: (json["messages"] as List).map((x) => Message.fromJson(x as Map<String, dynamic>)).toList());
+}
+
+class Message {
```

Comment:
```suggestion
class TxBodyMessageJson {
```
also, should be in separate file